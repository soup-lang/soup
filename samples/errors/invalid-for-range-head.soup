for i 10 a
    console.log(i)
console.log(10)
