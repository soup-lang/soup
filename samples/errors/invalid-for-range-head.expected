samples/errors/invalid-for-range-head.soup:1
    unexpected token: 10
